cookbooks/aws-parallelcluster-config/templates/default/directory_service/sssd.conf.erb

```diff
@@ -0,0 +1,38 @@
+[domain/default]
+debug_level = 0x1f0
+id_provider = ldap
+cache_credentials = True
+ldap_schema = AD
+ldap_uri = <%=  node['cluster']["directory_service"]["domain_addr"] %> # TODO: use ldaps
+ldap_search_base = <%=  node['cluster']["directory_service"]["domain_name"] %>
+ldap_default_bind_dn = <%=  node['cluster']["directory_service"]["domain_read_only_user"] %> # TODO: parametrize username
```

Comment:
is the TODO at the end of this line taken care of now?

Reply:
No, what does parametrize user name mean?

Reply:
we discussed this offline. it was a reference to an implementation detail of the PoC